### PR TITLE
GRAU-186

### PR DESCRIPTION
**Pull request by DANLENEHAN**

### CODE REVIEW

File: src/navigation/types.ts [hunk 1]
```diff
@@ -10,5 +10,9 @@ export type RootStackParamList = {
   App: undefined;
   Home: undefined;
   Login: undefined;
+  AccountSettings: undefined;
```
Comment (DanMaguireData) [inline src/navigation/types.ts:13]: If you merge master into this, I actually changed the structure of the typing

File: src/components/NavBar/Header.tsx [hunk 2]
```diff
@@ -21,11 +21,14 @@ import {RootStackParamList} from '../../navigation/types';
 type HeaderProps = {
   navigation: NativeStackNavigationProp<RootStackParamList>;
   label: string;
-  //targetScreen?: keyof RootStackParamList;
-  goBack?: boolean;
```
Comment (DanMaguireData) [inline src/components/NavBar/Header.tsx:25]: Just tell me you hate goBack d

Comment (DANLENEHAN) [reply src/components/NavBar/Header.tsx:25]: i'd done that before and saw the update in the conflicts and thought 'keep existing'

File: src/navigation/types.ts (hunk 1 above)
Comment (DanMaguireData) [reply src/navigation/types.ts:21]: This man chose violence putting these in here

Comment (DANLENEHAN) [reply src/navigation/types.ts]: tbf it was a bad merge didn't really see what was going on above

**Review by DanMaguireData — APPROVED** [review]

**Pull request merged**